test/integration/default/serverspec/default_spec.rb

```diff
@@ -36,8 +36,12 @@
         expect(retention_policies).to match(/default/)
       end
 
-      it 'is set for 1 week' do
-        expect(retention_policies).to match(/default.*168h0m0s/)
+      it 'its retention is set for 1 week' do
+        expect(retention_policies).to match(/default\s+168h0m0s/)
+      end
+
+      it 'its shard duration is set for 24 hours' do
+        expect(retention_policies).to match(/default\s+.*\s+24h0m0s/)
```

Comment:
Ensures shard duration is set correctly.  As an example, retention policies are shown as follows:
```
name    duration shardGroupDuration replicaN default
----    -------- ------------------ -------- -------
default 72h0m0s  24h0m0s            1        true
```
Thus the regex looks for `<policy_name><whitespace><any-duration><whitespace><desired_shard_duration>`